src/index.html

```diff
@@ -22,6 +22,453 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="/">
+        <img
+          src="images/logo.png"
+          alt="electronics-shop-logo"
+          class="header-logo"
+        />
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link is-active"
+            >
+              Apple
+            </a>
+          </li>
```

Comment:
The class `is-active` should be added to the link that leads to the current page. In this case, it seems that it's the 'Apple' page. If it's not the current page, consider removing the `is-active` class.